kogitoq/kogito-common/src/main/java/life/genny/kogito/common/service/InitService.java

```diff
@@ -130,85 +131,10 @@ public void sendAllAttributes() {
 		}
 	}
 
-	/**
-	 * Send PCM BaseEntities.
-	 */
-	public void sendPCMs() {
-
-		log.info("Sending PCMs for " + userToken.getProductCode());
-
-		String productCode = userToken.getProductCode();
-		BaseEntity user = beUtils.getUserBaseEntity();
-		CapabilitySet userCapabilities = capMan.getUserCapabilities(user);
-
-		// get pcms using search
-		SearchEntity searchEntity = new SearchEntity("SBE_PCMS", "PCM Search")
-				.add(new Filter(Attribute.PRI_CODE, Operator.LIKE, "PCM_%"))
-				.setAllColumns(true)
-				.setPageSize(1000)
-				.setRealm(productCode);
-
-		log.info(jsonb.toJson(searchEntity));
-
-		List<BaseEntity> pcms = searchUtils.searchBaseEntitys(searchEntity);
-		if (pcms == null) {
-			log.info("No PCMs found for " + productCode);
-			return;
-		}
-
-		log.info("Sending " + pcms.size() + " PCMs");
-
-		// configure ask msg
-		QDataAskMessage askMsg = new QDataAskMessage();
-		askMsg.setToken(userToken.getToken());
-		askMsg.setReplace(true);
-		askMsg.setAliasCode("PCM_INIT_ASK_MESSAGE");
-
-		List<Ask> asks = pcms.stream()
-		.peek((pcm) -> log.info("Processing ".concat(pcm.getCode())))
-		.map((pcm) -> {
-			String questionCode = pcm.getValue("PRI_QUESTION_CODE", null);
-			if (questionCode == null) {
-				log.warn("(" + pcm.getCode() + " :: " + pcm.getName() + ") null PRI_QUESTION_CODE");
-				return null;
-			}
-
-			Question question = databaseUtils.findQuestionByCode(productCode, questionCode);
-			if(!question.requirementsMet(userCapabilities)) {
-				log.warn("[!] User does not meet capability requirements for question: " + questionCode);
-				return null;
-			} else {
-				log.info("Passed Capabilities check: " + CommonUtils.getArrayString(question.getCapabilityRequirements()));
-			}
-
-			Ask ask = qwandaUtils.generateAskFromQuestion(question, user, userCapabilities);
-			if (ask == null) {
-				log.warn("(" + pcm.getCode() + " :: " + pcm.getName() + ") No asks found for " + question.getCode());
-			}
-
-			return ask;
-		})
-		// filter all pcms set to null by the map (this stops us having to query for questionCode twice, saving processing
-		.filter((pcm) -> pcm != null) 
-		.collect(Collectors.toList());
-
-		askMsg.setItems(asks);
-
-		KafkaUtils.writeMsg(KafkaTopic.WEBDATA, askMsg);
-
-		// configure msg and send
-		QDataBaseEntityMessage msg = new QDataBaseEntityMessage(pcms);
-		msg.setToken(userToken.getToken());
-		msg.setReplace(true);
-		msg.setAliasCode("PCM_INIT_MESSAGE");
-
-		KafkaUtils.writeMsg(KafkaTopic.WEBDATA, msg);
-	}
-
 	public void sendDrafts() {
 
 		BaseEntity user = beUtils.getUserBaseEntity();
-		Ask ask = qwandaUtils.generateAskFromQuestionCode("QUE_DRAFTS_GRP", user, user);
+		Ask ask = qwandaUtils.generateAskFromQuestionCode("QUE_DRAFTS_GRP", user, user, new CapabilitySet(user), new ReqConfig());
```

Comment:
This should fetch a CapabilitySet from CapabilitiesManager#getUserCapabilities() or CapabilitiesManager#getEntityCapabilites(user) so that the user's capabilities are used when generating this ask. Are we still sending drafts on top of that?